### PR TITLE
Adding Kubeflow Contributor Summit slides

### PR DESCRIPTION
**Pull request by theadactyl**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/552)

### CODE REVIEW

File: content/docs/about/events.md [hunk 1]
```diff
@@ -10,13 +10,59 @@ Please raise a [GitHub issue](https://github.com/kubeflow/website/issues/new) if
...
+      - [Demo video](https://drive.google.com/a/kubeflow.org/file/d/1ITkcf9YvYDJ9KiRlNWOHmMEFOe2DeMGi/view?usp=sharing)
+  - Examples, Codelabs, & Demos: Michelle Casbon, Google
+      - [Slides](https://docs.google.com/presentation/d/1h001JfBUaAjoPTrHpbuVRNJtrO1jWRMG_YRDzL2Pd3I/edit?usp=sharing)
 * [Data Day Texas, Austin](https://datadaytexas.com/), 26 January, 2019
   - [Kubeflow: Portable Machine Learning on Kubernetes](https://datadaytexas.com/2019/sessions#casbon): Michelle Casbon
       - [Slides](https://docs.google.com/presentation/d/1PvtKTw3KNbGurNymDddbsmEfLQd9cOfjbpDQ-2DSNd8)
 * [AI NEXTCon, Seattle](http://aisea19.xnextcon.com/), 23-26 January, 2019
   - [Build and Manage Machine Learning Pipelines](http://aisea19.xnextcon.com/talkabstract.html#ws-kubeflow): Amy Unruh
-* [KubeCon, Seattle](https://events.linuxfoundation.org/events/kubecon-cloudnativecon-north-america-2018/), 11-13 December, 2018
+* [KubeCon, Seattle](https://events.linuxfoundation.org/events/kubecon-cloudnativeco n-north-america-2018/), 11-13 December, 2018
```
Comment (richardsliu) [inline content/docs/about/events.md]: Extra space here?

File: content/docs/about/events.md [hunk 2]
```diff
@@ -10,13 +10,59 @@ Please raise a [GitHub issue](https://github.com/kubeflow/website/issues/new) if
 
 * [Google Cloud Next, San Francisco](https://cloud.withgoogle.com/next/sf), 9-11 April, 2019
   - [Undoing Human Bias at Scale With Kubeflow](https://cloud.withgoogle.com/next/sf/sessions?session=MLAI206): John Bohannon, Michelle Casbon
-* Kubeflow Contributor Summit 2019, Sunnyvale, CA, 12-13 March, 2019
+* [Kubeflow Contributor Summit 2019](https://docs.google.com/document/d/14Jr79aWVjsJg8xq38CLVE4rKRwM1_-gHLuxhGAhrGTI/edit?usp=sharing), Sunnyvale, CA, 12-13 March, 2019
+  - Kubeflow and the ML Landscape: Jeremy Lewi, Google
```
Comment (avdaredevil) [inline content/docs/about/events.md]: You could consider making this list into a table using the syntax:
```md
Team | Person | Slides / Other
--- | --- | ---
foo | bar | [slides](https://link/here)
```

Comment (avdaredevil) [inline content/docs/about/events.md]: You could consider making this list into a table using the syntax:
```md
Team | Person | Slides / Other
--- | --- | ---
foo | bar | [slides](https://link/here)
```

Comment (theadactyl) [reply content/docs/about/events.md]: Possible, but I was trying to go by the format used in the list -- would like to leave as is and fix all events with several talks in one refactor.

Comment (avdaredevil) [reply content/docs/about/events.md]: Sounds good!

**Pull request merged**